example-app/SantokuApp/src/features/demo-push-notification/pages/PushNotificationSenderPage.tsx

```diff
@@ -0,0 +1,325 @@
+        <SpecAndSourceCodeLink feature="push-notification" />
+        <Box flex={1} height={1} px="p16" backgroundColor="grey1" />
+        <StyledSpace height="p12" />
+        <Text>【このデバイスのプッシュ通知受信可否】</Text>
+        <Box p="p16">
+          {isFetchingAccountData ? (
+            <Text>プッシュ通知受信可否を判定中...</Text>
+          ) : isReceivableOnThisDevice ? (
+            <Text>このデバイスでプッシュ通知を受信可能です。</Text>
+          ) : (
+            <StyledColumn gap="p2">
+              <Text color="textRed" fontWeight="bold">
+                このデバイスではプッシュ通知を受信できません。
+              </Text>
+              {!(isAllowedPermission && hasFcmToken) && (
+                <StyledRow>
+                  <StyledTouchableOpacity onPress={navigateToPushNotificationStatus}>
+                    <Text color="blue">・設定を確認</Text>
+                  </StyledTouchableOpacity>
+                  <Text>してください。</Text>
+                </StyledRow>
+              )}
+              {!isRegisteredDeviceToken && (
+                <Text>・バックエンドAPIに存在しているアカウントで再ログインしてください。</Text>
+              )}
+            </StyledColumn>
+          )}
+        </Box>
+        <Box flex={1} height={1} px="p16" backgroundColor="grey1" />
+        <StyledSpace height="p12" />
+        <Text>プッシュ通知として送るパラメータを入力してください。</Text>
+        <Text>未入力の項目は、バックエンドに送信されません。</Text>
+        <StyledSpace height="p12" />
+        <SettingBox title="【共通設定】">
+          <WithLabel label="通知タイトル">
+            <StyledTextInput
+              value={form.values.notificationTitle}
+              onChangeText={form.handleChange('notificationTitle')}
+              borderBottomWidth={1}
+              multiline
+            />
+          </WithLabel>
+          <WithLabel label="通知ボディ">
+            <StyledTextInput
+              value={form.values.notificationBody}
+              onChangeText={form.handleChange('notificationBody')}
+              borderBottomWidth={1}
+              multiline
+            />
+          </WithLabel>
+          <StyledRow gap="p8" alignItems="center">
+            <Text>データ属性の追加</Text>
+            <Pressable onPress={addDataField}>
+              <AddIllustration color="blue" />
+            </Pressable>
+          </StyledRow>
+          <Text>{`Keyが未入力の項目は、バックエンドに送信されません。
+Keyが入力されており、Valueが未入力の場合は送信されます。`}</Text>
+          <StyledColumn>
+            {form.values.data.map((d, index) => {
+              return (
+                <StyledRow key={`data-${index}`} gap="p24">
+                  <Box flex={2}>
+                    <StyledTextInput
+                      value={d.key}
+                      onChangeText={form.handleChange(`data[${index}.key]`)}
+                      borderBottomWidth={1}
+                      placeholder="Key"
+                    />
+                  </Box>
+                  <Box flex={2}>
+                    <StyledTextInput
+                      value={d.value}
+                      onChangeText={form.handleChange(`data[${index}.value]`)}
```

Comment:
``form.handleChange(`data[${index}.value]`)`` でも動くのですが、
``form.handleChange(`data[${index}].value`)`` のほうが自然です。

※ どちらでも動く理由: lodash の toPath の仕様による。

```suggestion
                      onChangeText={form.handleChange(`data[${index}].key`)}
                      borderBottomWidth={1}
                      placeholder="Key"
                    />
                  </Box>
                  <Box flex={2}>
                    <StyledTextInput
                      value={d.value}
                      onChangeText={form.handleChange(`data[${index}].value`)}
```